distribution/lib/Standard/Table/0.0.0-dev/src/Internal/Split_Tokenize.enso

```diff
@@ -1,11 +1,12 @@
 from Standard.Base import all
 
+import Standard.Base.Data.Text.Regex.Regex
```

Comment:
Not needed as in `from Standard.Base import all`.

Reply:
Done